mypyc/irbuild/ll_builder.py

```diff
@@ -768,8 +775,7 @@ def translate_special_method_call(self,
         """
         ops = method_ops.get(name, [])
         call_c_ops_candidates = c_method_call_ops.get(name, [])
-        call_c_op = self.matching_call_c(call_c_ops_candidates, [base_reg] + args, line,
-                                         result_type=result_type)
+        call_c_op = self.matching_call_c(call_c_ops_candidates, [base_reg] + args, line)
```

Comment:
Result type seems important to preserve -- we may need it to box/unbox the result?

Reply:
Yes, you are right. I misunderstand this, I thought that return type in the description provided enough information but it's clearly not the case.